playbooks/roles/streisand-mirror/vars/tor.yml

```diff
@@ -12,40 +12,92 @@ tor_base_download_url: "https://dist.torproject.org/torbrowser"
 # Windows
 tor_browser_bundle_windows_filename: "torbrowser-install-{{ tor_browser_bundle_version }}_en-US.exe"
 tor_browser_bundle_windows_sig_filename: "{{ tor_browser_bundle_windows_filename }}.asc"
+
+tor_browser_bundle_windows_filename_fr: "torbrowser-install-{{ tor_browser_bundle_version }}_fr.exe"
```

Comment:
It seems like (at least for Tor) the difference in these URLs is just the suffix: `"_en-US"` or `"_fr"`.

Is there any way to make this diff a little bit smaller? With each of these vars having a `_fr` equiv I'm worried if we added three more languages then each var would be duplicated three more times.